samples/MvcSample.Web/Controllers/FormatFilterController.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace MvcSample.Web.Controllers
+{
+    public class FormatFilterController : Controller
```

Comment:
nit: When we show API controllers they should use Attribute routes, because we really do not support or recommend conventional routes for APIs